Dockerfile

```diff
@@ -8,11 +8,10 @@ RUN \
   apt-get clean && \
   rm -rf /var/lib/apt/lists/*
 
-ADD requirements.txt /app/
-RUN pip install --upgrade pip
-RUN pip install -r /app/requirements.txt
-RUN pip install -r /app/requirements_dev.txt
-
+# Install Python dependencies
+RUN curl https://raw.githubusercontent.com/kennethreitz/pipenv/master/get-pipenv.py | python
```

Comment:
It would be nice to add a comment explaining why we're not running `pip install pipenv` here (assuming that's the preferred way).